packages/search/src/style.scss

```diff
@@ -198,6 +198,23 @@ $input-z-index: 20;
 			display: block;
 		}
 
+		// Removes the fade overlay when the search is open and focused
+		&.has-focus {
+			.search-component__input-fade {
+
+				&::before {
+					display: none;
+				}
+
+				&.ltr {
+					/*!rtl:ignore*/
+					&::before {
+						display: none;
+					}
+				}
```

Comment:
We can drop this. `.ltr` is part of the selector and so if `.search-component__input-fade:before` is already `display:none`, it will inherit this condition. Could you test that?

Reply:
This is true, but we need to move the whole `&.has-focus` declaration below `.search-component__input-fade` (line 218). Because CSS 🤷 😄

Reply:
> Can we drop this declaration? `.ltr` is part of the selector, so if `.search-component__input-fade:before` is already `display:none`, theoretically adding `.ltr` should not yell any difference. Could you test that?

I tested this, It doesn't work if we remove the `.ltr` selector.